torch/csrc/distributed/rpc/rref_impl.cpp

```diff
@@ -236,6 +236,7 @@ const IValue& OwnerRRef::getValue() const {
       !getTimedOut(),
       "RRef creation via rpc.remote() timed out, and it "
       "is possible that the RRef on the owner node does not exist.");
+  // FIXME Should the wait be non-blocking? Or should we expose the flag too?
   future_->wait();
```

Comment:
nothing prevents RRefs to hold CUDA tensors, so technically, this should be blocking. However, since this has implications on perf, shall we run some internal benchmark to verify there is no perf regressions?